packages/manager/modules/emailpro/src/dashboard/emailpro.controller.js

```diff
@@ -206,6 +206,15 @@ export default /* @ngInject */ function EmailProCtrl(
           loadPtrTooltip($scope.exchange);
           loadPtrv6Tooltip($scope.exchange);
         }
+        if ($scope.exchange.isMXPlan) {
+          EmailPro.retrieveMxPlan()
+            .then(({ isZimbra }) => {
+              $scope.isMigratedMxPlan = $scope.exchange?.isMXPlan && isZimbra;
```

Comment:
```suggestion
              $scope.isMigratedMxPlan = isZimbra;
```
as the if indicated `isMxPlan === true`